pkg/mark/link.go

```diff
@@ -0,0 +1,69 @@
+package mark
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/kovetskiy/mark/pkg/confluence"
+)
+
+type Link struct {
+	MDLink string
+	Link   string
+}
+
+// ResolveRelativeLinks finds links in the markdown, and replaces one pointing to other Markdowns either by own link (if not created to Confluence yet) or witn actual Confluence link
+func ResolveRelativeLinks(
+	api *confluence.API,
+	markdown []byte,
+	base string,
+) ([]Link, error) {
+	var (
+		links []Link
+	)
+	re := regexp.MustCompile("\\[.*?\\]\\((.*?)\\)")
```

Comment:
Isn't this regexp too greedy? I see you are using `.*`

Reply:
It could be made a bit less greedier, but I don't know it it's reasonable to modify as it still stops at first closing parenthesis (both normal and square). 

FIrst part in square brackets can be "anything" (as it's just text), so only latter part is something that might have better regexps.